redalert/checks/run_bash_script.py

```diff
@@ -0,0 +1,35 @@
+# type: run-bash-script
+
+import subprocess
+import tempfile
+
+from .exc import CheckFailure
+
+class RunBashScript:
+    """A check for running a bash script and optionally checks output.
+
+    Takes the following arguments:
+    source (required): The source code of the script.
+    output: expected output string.
+    interpreter: path to bash. Default is '/bin/bash'.
```

Comment:
nit: I would write this as "path to shell used to run the script. Default is '/bin/bash'"